### PR TITLE
Init

### PR DESCRIPTION
**Pull request by goruha**
## What
* Init bastion tf module

## Why
* Trying to module tf code

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+  tags {
+    Name      = "${null_resource.default.triggers.id}"
+    Namespace = "${var.namespace}"
+    Stage     = "${var.stage}"
+  }
+}
+
+module "dns" {
+  source    = "git::https://github.com/cloudposse/tf_hostname.git?ref=init"
```
Comment (osterman) [inline main.tf]: When do you want to pin these to a tag?

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    create_before_destroy = true
+  }
+}
+
+data "aws_route53_zone" "domain" {
+  zone_id = "${var.zone_id}"
+}
+
+data "template_file" "user_data" {
+  template = "${file("${path.module}/${var.user_data_file}")}"
```
Comment (osterman) [inline main.tf]: This won't let the user pass in their own user data file.

Comment (osterman) [inline main.tf]: The user should pass in the contents of the file.

**Comment by goruha** [comment]
Solved

**Pull request merged**